doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -868,6 +871,8 @@ An example of a ``wget`` command line for crawling ("recursive downloading") of
 .. code-block:: bash
 
   wget -r -e robots=off -nH --cut-dirs=3 --content-disposition https://demo.dataverse.org/api/datasets/24/dirindex/
+  # or
+  wget -r -e robots=off -nH --cut-dirs=3 --content-disposition https://demo.dataverse.org/api/datasets/:persistentId/dirindex?persistentId=doi:${PERSISTENT_ID}
```

Comment:
One example shows a specific id, the other shows as a variable; they should at least be consistent